### PR TITLE
SPA Angular CLI working with platformOS guide

### PR DESCRIPTION
**Pull request by maciej-webpassion**

### CODE REVIEW

File: app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+searchable: true
+---
+
+This guide will help you connect your Angular app (created with CLI) with platformOS. The goal is to have Angular app ready for local development communicating with instance by api via proxy.
+
+This tutorial has been based on Angular version 8.
+
+## Requirements
+
+- [Set up Platform Instance](https://documentation.platformos.com/get-started)
```
Comment (pavelloz) [inline app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid]:
```suggestion
- [Set up Platform Instance](/get-started)
```

File: app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+slug: /company
+layout_name: app
+---
+```
+
+Angular router will do rest of the job.
+
+### Step 4: Local and production paths
+
+If I needed to add some asset files like for example images or custom svg icons for [Angular Materials](https://material.angular.io/components/icon/overview) or maybe translation files like for [ngx-translate](https://www.codeandweb.com/babeledit/tutorials/how-to-translate-your-angular8-app-with-ngx-translate), I would definitely store it in web-app/src/assets/
```
Comment (pavelloz) [inline app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid]:
```suggestion
If I needed to add some asset files like for example images or custom svg icons for [Angular Materials](https://material.angular.io/components/icon/overview) or maybe translation files like for [ngx-translate](https://www.codeandweb.com/babeledit/tutorials/how-to-translate-your-angular8-app-with-ngx-translate), I would definitely store it in `web-app/src/assets/`
```

File: app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+It will append proper paths into index.html automatically.
+
+### Step 3: Routing and Redirects
+
+platformOS routing is based on liquid files (pages and layouts).
+
+I assume that Angular router will do his job, so I need to rewrite all urls into one file.
+
+I will do this with pages and layouts. [Learn More about Pages and Layouts](https://documentation.platformos.com/tutorials/pages/pages)
```
Comment (pavelloz) [inline app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid]:
```suggestion
I will do this with pages and layouts. [Learn More about Pages and Layouts](/tutorials/pages/pages)
```

File: app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+This tutorial has been based on Angular version 8.
+
+## Requirements
+
+- [Set up Platform Instance](https://documentation.platformos.com/get-started)
+- [Installed Angular CLI](https://angular.io/cli)
+
+## Steps
+
+Connecting Angular with platformOS is a six-step process. Angular CLI is based on webpack so some steps will be valid for other frameworks/builds based on webpack.
```
Comment (pavelloz) [inline app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid]: Probably worth also mentioning usage of TS, or .ts extension, or just use .js extension

File: app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+* company/about
+* company/terms-of-service
+* company/info/some-example-page
+
+For all these links only one page has to be created (let's call it company.html.liquid):
+
+```
+---
+slug: /company
```
Comment (pavelloz) [inline app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid]: Shouldnt this be just `company` with no `/`?

File: app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
```
Comment (pavelloz) [inline app/views/pages/tutorials/web-apps/spa-angular-cli-and-platformos.liquid]:
```suggestion
slug: company
```

**Review by pavelloz — APPROVED** [review]

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited, thank you!

**Comment by diana-lakatos** [comment]
Fixes #710

**Comment by diana-lakatos** [comment]
Please don't merge yet, @maciej-webpassion is discussing some aspects with @Slashek and will get back to us when the topic is ready to be merged. Thanks!

**Comment by Developer-DNM** [comment]
This is exciting!

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited new parts.

**Pull request merged**